app/src/main/AndroidManifest.xml

```diff
@@ -23,14 +23,17 @@
             android:name=".MainActivity"
             android:configChanges="orientation|screenSize" />
 
+        <activity
+            android:name=".LoginActivity"
+            android:configChanges="orientation|screenSize" />
+
         <meta-data
             android:name="com.bridgefy.sdk.API_KEY"
             android:value="ec174e51-a4ab-4e01-9908-fe53b2833622" />
 
         <activity
-            android:name=".LoginActivity"
```

Comment:
Was LoginActivity what @bentekkie had it as in his initial working state?

Reply:
Yes, this part was the part I am not sure about if it will cause errors later on going to LoginActivity.
If it is incorrect (probably is) let me know of the change or edit it if possible.

Reply:
We should probably test it before pulling it then no?

Reply:
Yes I agree, let's test it!